packages/core/root/core-root-server-internal/jest.integration.config.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+module.exports = {
+  preset: '@kbn/test/jest_integration_node',
+  rootDir: '../../../..',
+  roots: ['<rootDir>/packages/core/root/core-root-server-internal'],
+};
```

Comment:
It may be fine for the `root` domain because it already depends on all the other Core domains, but for consistency, we may want to move those with the other tests under `src/core/server/integration_tests`?